.github/workflows/test.yaml

```
@@ -95,3 +95,67 @@ jobs:
       - uses: ./.github/actions/setup-env
       - name: Run optimized tests
         run: tox -e optimized
+
+  mutation-test:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: [self-hosted-ghr, size-xl-x64]
```

This is the magic sauce that runs the job on a beefier machine.